dist2src/cli.py

```diff
@@ -365,5 +450,54 @@ def convert(ctx, origin, dest):
     ctx.invoke(apply_patches, gitdir=dest_dir)
 
 
+@cli.command("create-tag")
+@click.argument("tag", type=click.STRING)
+@click.argument("path", type=click.Path(exists=True, file_okay=False))
+@click.argument("branch", type=click.STRING)
+@log_call
+def create_tag(tag, path, branch):
+    repo = git.Repo(path)
+    n = sum(1 for commit in repo.iter_commits(branch)) - 3
+    repo.create_tag(tag, ref=f"{branch}~{n}")
+
+
+@cli.command("convert-with-prep")
+@click.argument("origin", type=click.STRING)
+@click.argument("dest", type=click.STRING)
+@log_call
+@click.pass_context
+def convert_with_prep(ctx, origin, dest):
+    """Convert a dist-git repository into a source-git repository, using
+    'rpmbuild' and running the build until %prep.
```

Comment:
```suggestion
    'rpmbuild' and running the build until (including) %prep.
```

Reply:
True, true, true.

Choose to reuse the phrasing from the `rpmbuild` docs, which should have the same meaning.